joern-cli/frontends/csharpsrc2cpg/src/main/scala/io/joern/csharpsrc2cpg/astcreation/AstForExpressionsCreator.scala

```diff
@@ -238,8 +246,14 @@ trait AstForExpressionsCreator(implicit withSchemaValidation: ValidationMode) {
 
   def astForObjectCreationExpression(objectCreation: DotNetNodeInfo): Seq[Ast] = {
     val dispatchType = DispatchTypes.STATIC_DISPATCH
-    val typeFullName = nodeTypeFullName(objectCreation)
-    val arguments    = astForArgumentList(createDotNetNodeInfo(objectCreation.json(ParserKeys.ArgumentList)))
+    val typeFullName = Try(createDotNetNodeInfo(objectCreation.json(ParserKeys.Type))).toOption match {
```

Comment:
I think it's best if we simply use `nodeTypeFullName(typenode)` for all of this? e.g.
`Try(createDotNetNodeInfo(objectCreation.json(ParserKeys.Type))).toOption.map(nodeTypeFullName).getOrElse(Defines.Any)`